test/mpi/autogen.sh

```diff
@@ -78,6 +86,9 @@ if test ! -e include/mtest_mpix.h ; then
     touch include/mtest_mpix.h
 fi
 
+# Create the benchmark tests
+create_benchmarks
```

Comment:
How about rename it to "generate_benchmarks"?

Reply:
Done.